toolkit/utilitynetworks/src/androidTest/java/com/arcgismaps/toolkit/utilitynetworks/TraceToolTestRunner.kt

```diff
@@ -46,6 +47,23 @@ open class TraceToolTestRunner(
     internal val traceState: TraceState
         get() = _traceState ?: throw IllegalStateException("trace state is not initialized")
 
+    private var _map: ArcGISMap = ArcGISMap(PortalItem(Portal.arcGISOnline(connection = Portal.Connection.Anonymous), itemId))
+    internal val map: ArcGISMap
+        get() = _map
+
+    private var _mapviewProxy: MapViewProxy = MapViewProxy()
+    internal val mapviewProxy: MapViewProxy
+        get() = _mapviewProxy
+
+    private var _graphicsOverlay: GraphicsOverlay = GraphicsOverlay()
+    internal val graphicsOverlay: GraphicsOverlay
+        get() = _graphicsOverlay
+
+    // Create a CountDownLatch to wait for the draw status change
+    private val _drawStatusLatch = CountDownLatch(1)
+    internal val drawStatusLatch
+        get() = _drawStatusLatch
+
```

Comment:
```suggestion
    internal val map: ArcGISMap = ArcGISMap(PortalItem(Portal.arcGISOnline(connection = Portal.Connection.Anonymous), itemId))
    internal val mapviewProxy: MapViewProxy = MapViewProxy()
    internal val graphicsOverlay: GraphicsOverlay = GraphicsOverlay()
    // Create a CountDownLatch to wait for the draw status change
    internal val drawStatusLatch = CountDownLatch(1)
```

I am sorry I didn't see this on Friday. It doesn't appear necessary to have backing fields for these internal properties, or computed getters.